iac-common/src/main/java/org/sonar/iac/common/extension/visitors/InputFileContext.java

```diff
@@ -60,12 +60,14 @@ public void reportIssue(RuleKey ruleKey, @Nullable TextRange textRange, String m
 
       issue.forRule(ruleKey).at(issueLocation);
 
-      secondaryLocations.forEach(secondary -> issue.addLocation(
-        issue.newLocation()
-          .on(inputFile)
-          .at(secondary.textRange)
-          .message(secondary.message)
-      ));
+      secondaryLocations.stream()
+        .filter(Objects::nonNull)
```

Comment:
Had to add this filter on the lowest level to don't commit null as secondary locations